awx/main/migrations/0186_hop_nodes.py

```diff
@@ -2,11 +2,14 @@
 
 import django.core.validators
 from django.db import migrations, models
+from django.test.utils import override_settings
 
 
-def set_peers_from_control_nodes_true(apps, schema_editor):
-    Instance = apps.get_model('main', 'Instance')
-    Instance.objects.filter(node_type='execution').update(peers_from_control_nodes=True)
+def automatically_peer_from_control_plane(apps, schema_editor):
+    with override_settings(IS_K8S=True):
```

Comment:
Shouldn't this just be `if settings.IS_K8S`? the override_settings is a test utility and shouldn't be used in non-test code. It's function is to override the existing value, which isn't what you want, and the models here shouldn't be running any custom logic that it could affect anyway.

Reply:
Thank you so much, Alan! I was actually working on testing this piece today. I will revise accordingly. Thank you again for your feedback!